pages/UI.php

```diff
@@ -1853,7 +1853,7 @@ function DisplayNavigatorGroupTab($oP)
 
 		///////////////////////////////////////////////////////////////////////////////////////////
 		
-		case 'swf_navigator': // Graphical display of the relations "impact" / "depends on"
+		case 'navigate': // Graphical display of the relations "impact" / "depends on"
```

Comment:
I would recommend keeping the old operation, as there might be some modules out there depending on this. This will allow us to add the depreciation in our migration notes, and remove this operation on the next major version.

```suggestion
		case 'swf_navigate': // SWF was removed in iTop 
		case 'navigate': // Graphical display of the relations "impact" / "depends on"
```

Reply:
Good suggestion!

Reply:
Maybe adding an `@depricated` doc as well?

Reply:
I did hesitate to add it : it suits very well in a function / class PHPDoc, but for just one line, I'm not sure ? But indeed it's a keyword we're looking before releasing :) Maybe add it in the inline comment like this :
```suggestion
		case 'swf_navigate': // @deprecated SWF was removed in iTop 
		case 'navigate': // Graphical display of the relations "impact" / "depends on"
```

Reply:
I'm not sure this works with phpdoc. Can you verify first?

Reply:
I don't think that the deprecated tag is handled by IDEs for anything else than a class or a function... I didn't find any reference though :/ I was remembering that deprecated usages are strike through since a few versions in PHPStorm but I didn't find any release note about this (I didn't take time to do an exhaustive search...)

You version is ok for me :)

Reply:
Done: e0079a4